portfolio/src/main/webapp/script.js

```diff
@@ -39,33 +28,114 @@ window.onload = loadComments();
 //Retrieves json comments from server 
 function loadComments(){
     
-    var commentsLimit = commentAmount();
+    const commentsLimit = commentAmount();
+    const order = document.getElementById("order").value;
 
-    fetch('/data?number=' + commentsLimit).then(response => response.json()).then((comments) => {
+    fetch('/data?number=' + commentsLimit + '&order=' + order).then(response => response.json()).then((comments) => {
       const allCommentsList = document.getElementById('comments-container');
       allCommentsList.innerHTML = '';
       
       for(var i = 0; i < commentsLimit; i++){
-        let singleComment = document.createElement('p');
-        singleComment.innerText = comments[i].comment;
-        allCommentsList.append(singleComment);
+        allCommentsList.append(createSingleComment[i]);
       }
     });
 }
 
+function createSingleComment(comment){
+    const commentDiv = document.createElement('div');
+
+    const nameTitle = document.createElement('h3');
+    nameTitle.innerHTML = "Name: " + sanitizeHTML(comment.name);
+    commentDiv.appendChild(nameTitle);
+
+    const commentContent = document.createElement('p');
+    const moodReceived = comment.mood;
+    if (moodReceived === "happy"){
+        commentContent.innerHTML = "Comment: " + sanitizeHTML(comment.comment) + "  😊 ";
+    } else if(moodReceived === "heart"){
+        commentContent.innerHTML = "Comment: " + sanitizeHTML(comment.comment) + "  😍 ";
+    } else if(moodReceived === "surprised"){
+        commentContent.innerHTML = "Comment: " + sanitizeHTML(comment.comment) + "  😯 ";
+    } else if(moodReceived === "sad"){
+        commentContent.innerHTML = "Comment: " + sanitizeHTML(comment.comment) + "  😥 ";
+    } else {
+            commentContent.innerHTML = "Comment: " + sanitizeHTML(comment.comment);
+    }
+    commentDiv.appendChild(commentContent);
+
+    const deleteCommentBtn = document.createElement("button");
+    deleteCommentBtn.innerHTML("<p> delete </p>");
+    deleteCommentBtn.addEventListener('click', () => {
+        deleteSingleComment(comment);
+    });
+    commentDiv.appendChild(deleteCommentBtn);
+
+    return commentDiv;
+}
+
+//Prevents HTML injection by removing markup from code
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

Space after the `//`
`// Prevents HTML injection ...`